Scoop/Controller/Ride Posting/InitialPostRideViewController.swift

```diff
@@ -40,6 +44,14 @@ class InitialPostRideViewController: PostRideViewController {
                 return
             }
             
+            NetworkManager.shared.currentRide.type = self.studentDriverButton.isSelected ? "Student driver" : "Taxi"
+            if let arrival = self.arrivalTextField.text, let departure = self.departureTextField.text, let arrivalID = self.arrivalLocationID, let departureID = self.departureLocationID {
```

Comment:
This is just personal preference, but adjust the spacing so that all the "let" statements are aligned